integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -316,4 +324,17 @@ object IndexServer extends ServerInterface {
       Array(new Service("security.indexserver.protocol.acl", classOf[ServerInterface]))
     }
   }
+
+  def startAgingFolders(): Unit = {
```

Comment:
Please rename this method to something like "scheduleCleanUpThread"

Reply:
done